doc/DeveloperGuide.md

```diff
@@ -88,6 +88,31 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Edit person
+
+**MSS**
+
+1.User request to list persons
+2.AddressBook shows a list of persons
+3.User request to change details of one of the person
+4.User provides the detail to change, with the new arguments
```

Comment:
The use case shows the interaction between the user and the program. So for each new bullet, you should alternate between the user and the program, ie if the user did something in step 3, step 4 should be about what is done by the program.